src/Function/Relation/Binary/Setoid/Equality.agda

```diff
@@ -0,0 +1,50 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Function Equality setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Level using (Level; _⊔_)
+open import Relation.Binary.Bundles using (Setoid)
+
+module Function.Relation.Binary.Setoid.Equality {a₁ a₂ b₁ b₂ : Level}
+  (From : Setoid a₁ a₂) (To : Setoid b₁ b₂) where
+
+  open import Function.Bundles using (Func; _⟨$⟩_)
```

Comment:
We don't usually indent after top level modules?

Reply:
Indeed - that's one small difference between stdlib and categories. Can fix.